bin/test

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+
+require_relative '../utils/diggit'
+
+def verify_version(program, expected, actual)
+  unless expected == actual
```

Comment:
Think I want exact version actually, given that it doesn't cost much to install the exact version and it gets rid of niggly inconsistency issues